### PR TITLE
Issue fixes for #19438 and #19437

### LINKED ISSUE #19437: [enterprise-4.3] Edit suggested in file serverless/interacting-serverless-apps.adoc
### Which section(s) is the issue in?
Verifying your serverless application deployment

### What needs fixing?
"Find the application host."

should be 

"Find the application URL" 

as it's the URL that output as part of the "oc get ksvc helloworld-go" command and the value being used in the next step in the next step with CURL command

### PR DESCRIPTION
**Pull request by abrennan89**
Fixes #19437 and #19438

### CODE REVIEW

File: modules/interacting-serverless-apps-http2-gRPC.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ To do this you must find the ingress gateway's public address and the applicatio
 . The ingress gateway's public address can be determined using this command:
 +
 ----
-oc -n istio-system get svc istio-ingressgateway
+$ oc -n knative-serving-ingress get svc kourier
 ----
 +
 The output will be similar to this example:
```
Comment (mgencur) [inline modules/interacting-serverless-apps-http2-gRPC.adoc:21]: We also need to change the output of this example. It should be something like this:
```
----
NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP                                                             PORT(S)                                                                                                                                      AGE
kourier   LoadBalancer   172.30.51.103   a83e86291bcdd11e993af02b7a65e514-33544245.us-east-1.elb.amazonaws.com   80:31380/TCP,443:31390/TCP   67m
----
```

**Comment by abrennan89** [comment]
Thanks @mgencur - updated the example. This this should be OK for QE approval now?

**Comment by mgencur** [comment]
Thanks.
QE Approved.

**Comment by mburke5678** [comment]
@abrennan89 LGTM

**Comment by abrennan89** [comment]
Thanks @mburke5678 - could go you ahead and merge this please? It applies for OCP 4.3 and will apply for 4.4 also.

**Pull request merged**